support/sier-codec/src/object.rs

```diff
@@ -151,6 +165,13 @@ impl<'s> Value<'s> {
     pub fn serialize(&self) -> Vec<u8> {
         match self {
             Value::Unit => Vec::new(),
+            Value::Bool(v) => {
+                if *v {
+                    Vec::from(1u8.to_le_bytes())
+                } else {
+                    Vec::from(0u8.to_le_bytes())
+                }
```

Comment:
You're duplicating the Vec::from and .to_le_bytes. I would do something like the following.

```rust
let byte: u8 = if *v { 1 } else { 0 };
Vec::from(byte.to_le_bytes())
```